compose/service.py

```diff
@@ -787,22 +786,22 @@ def pull(self, ignore_pull_failures=False):
                 log.error(six.text_type(e))
 
 
-class Net(object):
```

Comment:
Right, sorry, I meant to update those but forgot. Done. Only failures now are AUFS, not related to our code.